fellow-submissions/peter-rose.md

```diff
@@ -0,0 +1,148 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+
+<!---->
+   Firstly, const cannot change its value. Thats the purpose of it so that variable will never be able to reassign to a new value. Unlike
+   let and var which are able to be reassigned.
+   
+   Secondly, let and const hoist but you can't access them before the declaration is read by the console. Check out Questions 6 & 7
+   in this .md file for a good example of this.
+   
+   Thirdly, var can be accessible anywhere in function but let and const can only be accessible inside the block where you first declare them.
+
+**2. What will the following code log? Why?**
+    ```javascript
+    let a = 'outer';
+
+    function testScope() {
+      let a = 'inner';
+      console.log(a);
+    }
+
+    console.log(a);
+    testScope();
+    console.log(a);
+    ```
+    <!---->
+    console.log(a) will ALWAYS log as outer. The function called on line 15 is refering to the variable 'a' thats initialized within itself
+    so it'll always refer to that value which is inner.
+    //ouput    outer --> inner --> outer
+
+**3. What will the following code log? Why?**
+    ```javascript
+    function hello() {
+      let a = 'hello';
+    }
+
+    hello();
+    console.log(a);
+    ```
+    <!---->
+    Code will result in an error, because a is not defined. this is because let has a block scope so the value of a cannot be 
```

Comment:
Yes! One small thing to note is that variables declared with `var` are function scoped so this would even throw an error if the variable on line 2 was declared with `var`.